hybrid/csp_source.py

```diff
@@ -178,7 +178,7 @@ def tmy3_to_df(self):
         df.index.name = 'datetime'
         df.drop(date_cols, axis=1, inplace=True)
 
-        df.index = df.index.map(lambda t: t.replace(year=df.index[0].year))  # normalize all years to that of 1/1
+        # df.index = df.index.map(lambda t: t.replace(year=df.index[0].year))  # normalize all years to that of 1/1
```

Comment:
I had commented this out because it caused an issue in more recent versions of pandas. I'm uncommenting it out now, but I think I need to pin the version of pandas